assets/templates/404.tmpl

```diff
@@ -0,0 +1,2 @@
+> <@{{ .Message.Author.ID }}> Oups ! Je sais pas faire !
```

Comment:
We may want to write a full documentation to help users defining their own templates